funnel/models/event.py

```diff
@@ -28,7 +28,30 @@ def make_private_key():
     )
 
 
-class Event(BaseScopedNameMixin, db.Model):
+class ScopedNameTitleMixin(object):
```

Comment:
MRO woes: https://github.com/hasgeek/funnel/commit/a1a81a805f81f7aecc80211c452f42aa6cfa9833

Had to clean up like this across all projects because I hadn't realised it was left to right.